token-support-idporten/src/main/kotlin/no/nav/tms/token/support/idporten/authentication/config/RuntimeContext.kt

```diff
@@ -42,10 +46,22 @@ private fun createOAuth2ServerSettings(
         accessTokenRequiresBasicAuth = false,
         requestMethod = HttpMethod.Post,
         defaultScopes = listOf(Idporten.scope),
-        authorizeUrlInterceptor = { this.parameters.append("response_mode", "query") },
+        authorizeUrlInterceptor = createAuthorizeUrlInterceptior(securityLevel),
         accessTokenInterceptor = idportenClientInterceptor.appendClientAssertion
 )
 
+private fun createAuthorizeUrlInterceptior(securityLevel: SecurityLevel): URLBuilder.() -> Unit {
+    return {
+        parameters.append("response_mode", "query")
+
+        when (securityLevel) {
+            LEVEL_3 -> parameters.append("acr_values", "Level3")
+            LEVEL_4 -> parameters.append("acr_values", "Level4")
+            NOT_SPECIFIED -> {}
```

Comment:
Burdet vi kastet en feil her hvis vi ikke mottar enten nivå 3 eller 4? Kan ikke det være en indikator på at noe alvorlig er galt?

Reply:
Dette er gjort slik for å speile ID-porten, som tillater 3, 4 og ikke spesifisert.

Reply:
"NOT_SPECIFIED" vil egentlig si at api-et ikke stiller noen krav om det er nivå 3 eller 4, men det er fortsatt kunn de to som er aktuelle.